samples/scaleout/Version_6/Shared/ConfigErrorQueue.cs

```diff
@@ -0,0 +1,13 @@
+﻿using NServiceBus.Config;
+using NServiceBus.Config.ConfigurationSource;
+
+class ConfigErrorQueue : IProvideConfiguration<MessageForwardingInCaseOfFaultConfig>
```

Comment:
The V5 endpoints (sender and server) can't use that. Is there a benefit of moving the workers to new code API?